berg.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: berg
+  version: 0.3.5
+  epoch: 0
+  description: "CLI Tool for Codeberg similar to gh and glab"
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - build-base
+      - openssl-dev
+      - rust
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://codeberg.org/RobWalt/codeberg-cli/
+      tag: v${{package.version}}
+      expected-commit: b1850444bef36a16bb9512e73802d7ca2151a0bf
+
+  - name: Configure and build
+    runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      cargo build --all-features
+      install -m755 target/debug/berg "${{targets.destdir}}"/usr/bin/
```

Comment:
```suggestion
      install -m755 target/release/berg "${{targets.destdir}}"/usr/bin/
```

Reply:
When building with the changed path, melange throws an error

```
⚠️  x86_64    |     Finished dev [unoptimized + debuginfo] target(s) in 45.04s
⚠️  x86_64    | install: can't stat 'target/release/berg': No such file or directory
2023/09/08 21:29:42 ERROR: failed to build package. the build environment has been preserved:
ℹ️  x86_64    |   workspace dir: /tmp/melange-workspace-2690896317
ℹ️  x86_64    |   guest dir: /tmp/melange-guest-573171705
2023/09/08 21:29:42 error during command execution: failed to build package: unable to run pipeline: exit status 1
```
I just submitted updated build flags